api/loaders/graph_loader.py

```
@@ -49,7 +49,7 @@ async def load_to_graph(
             {"size": vec_len},
         )
         await graph.query("CREATE INDEX FOR (p:Table) ON (p.name)")
-    except Exception as e:
+    except Exception as e:  # pylint: disable=broad-exception-caught
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)